Add DrawSteelActiveEffectConfig

cdverrett94 opened the pull request:
Closes #226 

I left the core fields in for now even if they aren't useful, but if we want to remove them now, I can do that.

JPMeehan commented on src/module/apps/_module.mjs:
```diff
@@ -1,3 +1,4 @@
+export {DrawSteelActiveEffectConfig} from "./active-effect-config.mjs";
```
This should go in the `/core` folder

cdverrett94 replied:
Didn't realize that was there.

JPMeehan commented on src/module/apps/active-effect-config.mjs:
```diff
@@ -0,0 +1,42 @@
+ */
+export class DrawSteelActiveEffectConfig extends sheets.ActiveEffectConfig {
+
+  static DEFAULT_OPTIONS = {
```
```suggestion
  /** @override */
  static DEFAULT_OPTIONS = {
```

cdverrett94 replied:
ooof

JPMeehan commented on src/module/apps/active-effect-config.mjs:
```diff
@@ -0,0 +1,42 @@
+    classes: ["draw-steel"]
+  };
+
+  static PARTS = {
```
```suggestion
  /** @override */
  static PARTS = {
```

JPMeehan commented on src/module/apps/active-effect-config.mjs:
```diff
@@ -0,0 +1,42 @@
+    }
+  };
+
+  async _prepareContext(options) {
```
```suggestion
  /** @override */
  async _prepareContext(options) {
```

JPMeehan commented on src/module/apps/active-effect-config.mjs:
```diff
@@ -0,0 +1,42 @@
+import {systemPath} from "../constants.mjs";
+
+const {sheets} = foundry.applications;
```
Would prefer to drop the dereferencing since we're only extending one class instead of class + mixin

cdverrett94 replied:
Fair. I mostly did it because the line was pretty long already.

JPMeehan reviewed (changes requested):
I also had to double check that core is actually just using that inline `#if tab.active`, messy but would rather not convolute the context prep.

cdverrett94 commented:
> I also had to double check that core is actually just using that inline `#if tab.active`, messy but would rather not convolute the context prep.

I changed this to just use `{{tab.cssClass}}` since its in the context already. Initially I just copy/pasted the core duration template and just added our fields without looking at that piece.
